code-for-canada/project-thundercat
# [Org Charts] Updating english images of org charts

# Description

Uploading 2 new english images for the org charts.

## Type of change

- Code cleanliness or refactor

## Screenshot

General EN

![image](https://user-images.githubusercontent.com/2746350/54699649-ceecf200-4b07-11e9-9233-70e29dfe1620.png)
![image](https://user-images.githubusercontent.com/2746350/54699674-d8765a00-4b07-11e9-9137-b31516cf2965.png)

General FR
![image](https://user-images.githubusercontent.com/2746350/54703847-f8aa1700-4b0f-11e9-8a76-2e4361b712ef.png)
![image](https://user-images.githubusercontent.com/2746350/54703871-0495d900-4b10-11e9-9a00-cb1fba2bc7fd.png)

Team EN
![image](https://user-images.githubusercontent.com/2746350/54699705-e88e3980-4b07-11e9-8047-d2034eddf9d5.png)
![image](https://user-images.githubusercontent.com/2746350/54699719-f0e67480-4b07-11e9-9a1d-2f5bd827399e.png)

Team FR
![image](https://user-images.githubusercontent.com/2746350/54703912-1a0b0300-4b10-11e9-8ee5-6cf264b103aa.png)
![image](https://user-images.githubusercontent.com/2746350/54703962-327b1d80-4b10-11e9-9fe8-9e671fa1f3db.png)



# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Go to Protoype -> Start eMIB
2. Navigate through the pre-test instructions
3. Enter the test, verify that the EN versions of the org charts look new and look good when zoomed.

# Checklist

Applicable for all code changes.
